pkg/services/ngalert/schedule/schedule.go

```diff
@@ -355,10 +355,11 @@ func (sch *schedule) WarmStateCache(st *state.Manager) {
 		}
 		for _, entry := range cmd.Result {
 			lbs := map[string]string(entry.Labels)
+			cacheId, _, _ := entry.Labels.StringAndHash()
```

Comment:
This is doing a sha1 has as well, so update to get just the string.

Reply:
Isn't it better to use the hash?

Reply:
Could be, although not human readable in any way if debugging - and I already updated all those tests for the string :P